app/src/main/kotlin/com/wire/android/media/PingRinger.kt

```diff
@@ -0,0 +1,94 @@
+package com.wire.android.media
+
+import android.content.Context
+import android.content.Context.VIBRATOR_SERVICE
+import android.media.AudioAttributes
+import android.media.AudioManager
+import android.media.MediaPlayer
+import android.os.Build
+import android.os.VibrationEffect
+import android.os.Vibrator
+import android.os.VibratorManager
+import com.wire.android.appLogger
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class PingRinger @Inject constructor(private val context: Context) {
+
+    private var mediaPlayer: MediaPlayer? = null
+    private var vibrator: Vibrator? = null
```

Comment:
having these object nullable scares me a little! can we get rid of these nullability?